### PR TITLE
Reworks Clockcult's standard conversion method, removes Taunting Tirade

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
balance: Geis now mutes for 12-14 seconds and "stuns" the target, via a binding effect, for 25 seconds instead of initiating a conversion. 
experiment: Geis's "stun" restrains the target and prevents them from taking actions, but its duration is halved if the binding is not being pulled by a Servant.
wip: Using Geis on a target prevents you from taking actions other than attempting to pull the binding. If you are pulling the binding, you can dispel it at any time.
wip: As should be obvious, if the binding is destroyed or dispelled, you can once again take normal actions.
tweak: Sigils of Submission are now permanent and have been moved from the Script tier to the Driver tier, with an according cost adjustment. They still do not penetrate mindshield implants.
rscdel: Removed Taunting Tirade.
rscdel: Removed Sigils of Accession.
/:cl:

Hey look it's #29240 except I did it in a bit over an hour.

Effectively, you need to set up a sigil beforehand, then drag your target to it with Geis.
This adds some opportunities for people to fuck with you dragging someone off, since conversion isn't as fast and, if you're using Geis to drag someone off, if very obvious.

This is basically bloodcult flow, except the stun doesn't double as a way to murder the shit out of people without an ally, so both the stun and the mute are a bit longer; 25 seconds of stun instead of 20 and 12-14 seconds of mute instead of 8-10.

### tl;dr Xhuis made some handy dandy flowcharts, this is how stuff works now:
[Old Conversion](http://i.imgur.com/1chOQAL.png) | [New Conversion](http://i.imgur.com/tjdJVo3.png)

### CODE REVIEW

**Comment by Supermichael777** [comment]
bloodcult should be more like this

**Comment by Xhuis** [comment]
The reason I closed the earlier PR is because it wasn't necessary in the freeze. This is, uhh, not covered by that anymore.

**Comment by meeeeme** [comment]
So basically stunpaper?

**Comment by Xhuis** [comment]
Another thing, @ChangelingRain: you seem to be very strongly of the opinion that I'm just slower than you and should leave the coding to you. I want to clear the air there; the coding itself is easy and quick for me to do, but I'm very concerned about getting feedback on changes and deciding the "optimal" thing to do. I can easily make a huge change and PR it, but I'm very cautious when it comes to touching gamemodes.

You said that that's why you do things yourself, because it's quicker - I can do that too, but I like to tweak and poke at changes based on what players think of them. I'm not trying to bash you, of course; just give some reasons as to why I operate in the way I do.

**Comment by Repukan** [comment]
Freeze

**Comment by Repukan** [comment]
Can you two get a room and finish masturbating over this mode and just work on the full rework rather than keep trying to sneak past only balances under the pretense of "teehee they gave us a hand so we'll take the whole arm"

**Comment by Xhuis** [comment]
If you'd actually been paying attention instead of focusing on bashing us, you'd see that I'm actually beginning to work on the rework now that the mode is (in my opinion) in a decent state. Then again, I don't expect someone like you to look any deeper than the surface, so...

**Comment by Repukan** [comment]
Yeah, because you two idiots abused the good faith leeway that was given to *you* after you brought up the rework idea.

Clock cult has been the only mode that has received changes that are arguably more to balance it in its current state than actually move it closer to the so promised rework you keep mentioning. You know you could have done all those changes and made just one big commit to wrap up the rework, no?

Instead you post meme seal pictures while more "balance" changes keep getting added with no real sign of the so promised rework being shown.

**Comment by Repukan** [comment]
In the end the only thing we can see is what you show, and what you showed was that the small leeway you were given was more likely (from some people's perception) abused. 

If you had posted at least any commits or something like a progress on how the changes are coming, then it wouldn't look like you're abusing said leeway

**Comment by Xhuis** [comment]
I spoke with Kor about that. All three of us were operating under the assumption that the changes we were making _were_ allowed by the freeze, because he had phrased something oddly. This was also said, but you seem to have no missed the memo.

**Comment by Supermichael777** [comment]
For a man who never contributes anything repukan sure posts a lot

**Comment by Repukan** [comment]
You don't need to be a chef to know something tastes like shit

**Comment by ChangelingRain** [comment]
That'd be a better response if you had actually been attacking the implementation... which you weren't.

**Comment by lzimann** [comment]
@KorPhaeron

**Comment by Shadowlight213** [comment]
@ChangelingRain flowchart lacks arrows on connections

**Comment by Shadowlight213** [comment]
@Xhuis flowchart lacks arrows on connections

**Comment by GuyonBroadway** [comment]
I still think Clock Cult needs to move away from blood cult. Not towards.

**Comment by KingPhilipIII** [comment]
I'm still against this because I have inherent issues with moving closer to another antagonist's basic mechanics of play.

**Comment by Tacolizard** [comment]
FREEZE

**Comment by Tacolizard** [comment]
this is now a designated frozoneposting PR

**Comment by AdamElTablawy** [comment]
forgive me for asking but this is moving slowly closer to being bloodcult, isn't it?

we went from having an AOE convert and being able to convert mindshielded people to having a single-person stun convert or being forced to use a sigil that sometimes converts mindshielded people depending on what sigil it is and now we're at single-target-cheap-stun and using a sigil that converts non-mindshielded people again.

so we have a stunpaper analogue which stuns for longer but is obvious and a convert rune analogue that can be used by a single person and takes longer.

**Comment by KorPhaeron** [comment]
@Xhuis why did you downvote this

**Comment by Xhuis** [comment]
It was opened after your statement that clockcult PRs not pertaining to the rework would be shut down. I don't feel as though it's necessary right now. @ChangelingRain has told me that she's deadset on making her own vision of the mode, so I'm questioning if the rework is even worth pursuing if it'll just cause trouble.

**Comment by Repukan** [comment]
Punished Xhuis: A coder cucked out of his own gamemode.

**Comment by Xhuis** [comment]
Clockcult is a cycle of coders being cucked out of their gamemode. First Velard made it, and then he got cucked out of it by PJB. Then PJB made it and I cucked him out if it. I then willingly allowed Joan to cuck me out of it, and now I'm attempt to re-cuck it back with the rework, but only if the maintainers allow it to be cucked.

**Comment by Repukan** [comment]
😂

**Comment by drfino** [comment]
Isn't this basically the same as blood cult? Paper = geis, offer rune = sigil...? This isn't really any different from that other geis PR from before. I don't see the point of having two cults if they play mostly the same way. Read through some of the comments in the other thread since they still apply here.

**Comment by ChangelingRain** [comment]
It's much better if they play a little similarly in regards to conversion, since it's a lot more confusing if they're extremely different.
Also clockcult's current conversion mechanics are confusing and badly-explained...

**Comment by drfino** [comment]
I disagree. Why have two antags with similar play styles at all? If they should play similarly, you might as well just get rid of one, or have it as a reskin. Making them play the same removes the significance of having two different cults antags (other than the deep lore).

**Comment by RussellTalbott** [comment]
_in Xhuis voice_ please date me Joan

**Comment by Xhuis** [comment]
Dating would make the rework more complicated, so I pass.

**Comment by KorPhaeron** [comment]
>@ChangelingRain has told me that she's deadset on making her own vision of the mode

I thought the whole justification for these being open during the freeze was that she was helping you rework it though

**Comment by Xhuis** [comment]
The reason I was allowing some PRs through is because they improved the state of the mode as it was, and would be thus helpful for that rework. While I'm not sure if that was the original way she saw it, she's stated that she doesn't see it that way now and doesn't really seem to care about what I say.

**Comment by ChangelingRain** [comment]
@drfino they don't play very similarly, though, and besides this I'm aiming towards making them more distinct, instead of less.

Conversion being 'drag man onto circle' is really as similar as it gets, since both do it differently as far as the actual mechanics of stunning and conversion go:
Cult's stun is a resource that they have to put time in to renew, and lets them also murder people solo, while this clockcult stun is an infinite resource but doesn't let a single person murder whoever they hit with it or stun a single person multiple times(without allies).
Cult's conversion requires a second person, but is instant, while this version of clockcult conversion just requires time rather than manpower.

**Comment by GuyonBroadway** [comment]
Interestingly I feel like it should be the other way around. Clock cult is all about bade building and they should have the static conversion rune where blood cult shoukd be more mobile and stealthy in their conversions.

**Comment by drfino** [comment]
@GuyonBroadway I agree 100%. I didn't like the removal of the channeled stationary spells because it's such an interesting mechanic that fit the clock theme so well. Nobody used them but they were actually the most clock-cult-like spells they had. I'd rather see reworks of existing under-used features than just outright removal of them.

**Comment by ChangelingRain** [comment]
I don't disagree that they're fitting, but they're also either mostly unusable since you can be interrupted by pushing and consume components regardless of 'success' or very strong if you buckle to a stationary object or otherwise become immovable, possibly by hiding behind stuff.

**Comment by drfino** [comment]
I'd like to see the channeled spells added back in, and if you're casting them over a clockwork tile, you are immovable. Promotes base building and defense. Stuff like that should be buffed not removed imo

**Comment by GuyonBroadway** [comment]
This was met with mostly negative feedback and I see no discussion about it on the forums. Why was it merged?

**Comment by Repukan** [comment]
Because xhuis got cucked out of his own gamemode again.

**Comment by drfino** [comment]
Also, isn't this the same change that was shot down last week? #29240

**Comment by GuyonBroadway** [comment]
None of the critique was even properly addressed, this was just merged with little to no visible process.

**Comment by PopNotes** [comment]
@GuyonBroadway welcome to /tg/ github, the fucking mad max of githubs
